projects/packages/publicize/src/jetpack-social-settings/class-settings.php

```diff
@@ -206,6 +208,25 @@ public function get_initial_state() {
 		return $settings;
 	}
 
+	/**
+	 * Get the list of supported Publicize services.
+	 *
+	 * @return array List of external services and their settings.
+	 */
+	public function get_services() {
+		$site_id = Manager::get_site_id();
+		if ( is_wp_error( $site_id ) ) {
+			return [];
+		}
+		$path     = sprintf( '/sites/%d/external-services', $site_id );
+		$response = Client::wpcom_json_api_request_as_user( $path );
+		if ( is_wp_error( $response ) ) {
+			return [];
+		}
+		$body = json_decode( wp_remote_retrieve_body( $response ) );
+		return array_values( (array) $body->services ) ?? [];
```

Comment:
We can return only the publicize services.

```suggestion
		return array_values(
			array_filter(
				(array) $body->services ?? array(),
				function ( $service ) {
					return isset( $service->type ) && 'publicize' === $service->type;
				}
			)
		);
```